content/en/docs/07/_index.md

```diff
@@ -328,7 +328,7 @@ show tables;
 
 ## Task {{% param sectionnumber %}}.4: Import a database dump
 
-Our task is now to import this [dump.sql](https://raw.githubusercontent.com/acend/kubernetes-basics-training/master/content/en/docs/07/dump.sql) into the MariaDB database running as a Pod. Use the `mysql` command line utility to do this. Make sure the database is empty beforehand. You could also delete and recreate the database.
+Our task is now to import this [dump.sql](https://raw.githubusercontent.com/acend/kubernetes-basics-training/master/content/en/docs/07/dump.sql) into the MariaDB database running as a Pod. Use the `mysql` command-line utility to do this. Make sure the database is empty beforehand. You could also delete and recreate the database.
```

Comment:
Don't agree. See e.g. https://www.computerhope.com/jargon/c/commandi.htm

Reply:
What about https://github.com/acend/kubernetes-basics-training/blob/master/content/en/setup/_index.md#command-line-interface-tool?

Reply:
I guess you mean the use of "command-line" in that file? In my opinion that should be "command line" as well. It's my commit because I simply copied the setup instructions and missed to fix it.